docs/lambda.md

```diff
@@ -0,0 +1,121 @@
+<!--
+SPDX-FileCopyrightText: 2021 Tezos Commons
+SPDX-License-Identifier: LicenseRef-MIT-TC
+-->
+
+# Lambda DAO
+
+LambdaDAO allows dynamic configuration of proposals and proposal handlers,
+via proposals. This allows amending the logic of handling different kinds of
+proposals, or even adding or removing proposal types.
+
+LambdaDAO contains three concrete proposal types, which are "add handler", "remove handler"
+and "execute handler". The "add handler" proposal type can be used to defined a new
+proposal handler and "execute handler" proposal type can be used to invoke an existing
+proposal handler.
```

Comment:
A bit odd that there is no explanation of "remove handler", however TBH I would remove this sentence entirely and instead say that they are described below.

Reply:
Have removed the section.